infra.bs

```diff
@@ -296,30 +296,30 @@ for working with them, in order to create common ground.
 <h3 id=lists>Lists</h3>
 
 <p>A <dfn export>list</dfn> is a specification type consisting of a finite ordered sequence of
-items.
+<dfn for="list,stack,queue,set" lt=item export>items</dfn>.
 
 <p>For notational convenience, a literal syntax can be used to express <a>lists</a>, by surrounding
-the list contents by « » characters and separating list items with a comma. An indexing syntax can
-be used by providing a zero-based index into a list inside square brackets.
+the list contents by « » characters and separating list <a for=list>items</a> with a comma. An
```

Comment:
I wonder if this is not better phrased as "surrounding its [items] by « » characters and separating them with a comma"